README.md

```diff
@@ -5,86 +5,94 @@ A tiny Express wrapper for safe, rapid microservice development
 ## installation
 
 ```bash
-npm install express
-npm install body-parser
+npm install express@5
 npm install @apostrophecms/minuscule
 ```
 
 ## usage
 
 ```javascript
-const express = require('express');
-const { WebError, minuscule } = require('@apostrophecms/minuscule');
-const app = express();
-const bodyParser = require('body-parser');
-const yup = require('yup');
+import express from 'express';
+import yup from 'yup';
+import { WebError, minuscule } from '@apostrophecms/minuscule';
 
-// Example yup schema, see yup documentation
-const projectSchema = object({
-  shortName: string().required(),
-  longName: string(),
-  prod: boolean()
-});
+const app = minuscule(express());
 
 // Allow traditional form submission format (if you want it)
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.urlencoded({ extended: false }));
 
 // Allow JSON submissions (recommended)
-app.use(bodyParser.json());
+app.use(express.json());
 
-const {
-  get,
-  post,
-  use
-} = minuscule(app);
+app.use(expectApiKey);
 
-use(expectApiKey);
+// Example yup schema, see yup documentation
+const projectSchema = object({
+  shortName: string().required(),
+  longName: string(),
+  prod: boolean()
+});
 
 // async GET API functions that just return a result
 
-get('/projects/:projectId', expectProjectId, async req => {
+get('/projects/:projectId', expectProjectId, async (req, res, next) => {
   const result = await myDatabase.findOne({
     id: req.params.projectId
   });
-  // returning an object -> automatic JSON response via req.res
-  return result;
+
+  return res.send(result);
```

Comment:
The sole purpose of minuscule is to remove cognitive load by removing the need to ever think about `res`, at least for typical cases like returning a value or throwing an error. You just return a value. Intuitive, like writing a function. Because APIs are essentially remote function calls.

This reduces bugs by eliminating the risk of failing to handle `res`.

Changing this back to req, res, next defeats the purpose of having `minuscule`. It also would defeat the purpose of having `apiRoutes` in apostrophe, which inspired `minuscule`.